src/MotoROS2_AllControllers.vcxproj

```diff
@@ -126,109 +126,109 @@
     <NMakeBuildCommandLine>"$(MP_VS_Install)mpBuilder.exe" -c $(Configuration) -p "$(ProjectDir)\" -n "mr2_dx2_h" -b "$(SolutionDir)bin\$(Configuration)" -o build -i "$(MP_VS_Install)DX200\inc"</NMakeBuildCommandLine>
```

Comment:
I thought I'd had to update this as well to include the `$(SolutionDir)lib` directory as an include directory, but for some reason that's not needed.

Reply:
The `lib` subfolder is being automatically passed to the compiler executable with a `isystem` tag. 

I'm sure it's due to the shenanigans that mpBuilder is doing to parse all the subfolders. But I don't fully understand it.

Reply:
Oh, I think it's pulling the path out of the vcxproj file